aospy/test/test_calc_basic.py

```diff
@@ -160,6 +161,45 @@ def setUp(self):
         }
 
 
+test_params = {
+    'proj': example_proj,
+    'model': example_model,
+    'run': example_run,
+    'var': var_not_time_defined,
+    'date_range': 'default',
+    'intvl_out': 1,
+}
+
+
+@pytest.mark.parametrize(('dtype_out_time'), [None, 'None', []])
+def test_calc_object_no_time_options(dtype_out_time):
+    test_params['dtype_out_time'] = dtype_out_time
+    calc = CalcInterface(**test_params)
+    if isinstance(dtype_out_time, list):
+        assert calc.dtype_out_time == tuple(dtype_out_time)
+    else:
+        assert calc.dtype_out_time == tuple([dtype_out_time])
+
+
+@pytest.mark.parametrize(
+    ('dtype_out_time'),
+    ['av', 'std', 'ts', 'reg.av', 'reg.std', 'reg.ts'])
+def test_calc_object_string_time_options(dtype_out_time):
+    test_params['dtype_out_time'] = dtype_out_time
+    with pytest.raises(ValueError):
+        CalcInterface(**test_params)
+
+
+def test_calc_object_time_options():
+    time_options = ['av', 'std', 'ts', 'reg.av', 'reg.std', 'reg.ts', 'None']
```

Comment:
Per @spencerahill's earlier comment, should we use `None` rather than `'None'` here?

Reply:
https://github.com/spencerahill/aospy/blob/67ec6197a97a13dfe7a63ad7a555dcc9056bd8c8/aospy/calc.py#L486, I thought `'None'` was a possible time reduction.

Reply:
Good catch @micahkim23!  @spencerahill was this intentional?  Should we change it?

Reply:
Woops!  Yes, good call @micahkim23 .  Sorry if any comments I made previously were confused on this point.

Since `'None'` appears to just be a synonym for `'ts'`, I'd say we just remove it entirely.  So there's only `['ts', 'av', 'std', 'reg.ts', 'reg.av', 'reg.std']`.

Since custom reductions are a big priority moving forward (#208), this will get refactored soon-ish anyways.